cmd/version.go

```diff
@@ -0,0 +1,23 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	version = "unknown"
```

Comment:
Does this need to be updated in the Makefile build?

Reply:
8ea4ff3 fixes this

Reply:
:+1: LGTM